clipmenud

```diff
@@ -121,6 +122,12 @@ while true; do
         fi
     fi
 
+    windowname="$(xdotool getactivewindow getwindowname)"
+    if [[ -n "${CM_IGNORE_WINDOW+x}" && "$windowname" =~ $CM_IGNORE_WINDOW ]]; then
```

Comment:
You don't need `+x` or `-n` here

Reply:
I wanted to distinguish between `CM_IGNORE_WINDOW` being unset and `CM_IGNORE_WINDOW` being set to the empty string, so I used the approach from https://stackoverflow.com/questions/3601515/how-to-check-if-a-variable-is-set-in-bash (I thought the empty string would match the empty string and not be treated as a wildcard, so it would be helpful to include). I'll update the code